static/src/js/components/SpatialDisplay/SpatialDisplay.js

```diff
@@ -359,9 +359,17 @@ class SpatialDisplay extends Component {
    * @param {Array} boundingBox Array with [swPoint, nePoint] values
    */
   validateBoundingBoxCoordinates(boundingBox) {
+    let errorMessage = ''
     const [swPoint, nePoint] = boundingBox
 
-    return this.validateCoordinate(swPoint) + this.validateCoordinate(nePoint)
+    errorMessage = this.validateCoordinate(swPoint) + this.validateCoordinate(nePoint)
+
+    if (swPoint === nePoint) {
```

Comment:
So if a user has both an invalid coordinate and they are the same coordinates what does the error message look like its just both of those combined right?

Reply:
Yes sir!